src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -29,6 +29,12 @@ const propTypes = {
     // eslint-disable-next-line react/forbid-prop-types
     fullReport: PropTypes.object,
 
+    /** The policies which the user has access to and which the report could be tied to */
+    policies: PropTypes.shape({
+        /** Name of the policy */
+        name: PropTypes.string,
+    }),
+
     ...withCurrentReportIDPropTypes,
```

Comment:
policies is an object with the shape { policyID: object }. In addition, we use the avatar of the policy as well. Can you explain?

Reply:
You are right, I was typing that incorrectly, fixed it!